docs/legacy-version-migration.md

```diff
@@ -0,0 +1,46 @@
+---
+hide:
+  - navigation
+---
+
+This documentation page lists the migration paths of Taipy releases as they
+were published.
+
+## From 2.0 to 2.1
+
+In Taipy version 2.1, the version management system has been introduced. For
+applications created with a Taipy Core version &#8804 2.0, the first time it
+runs with version 2.1 or later, no version exists, and so legacy entities are not
+attached to any version. Depending on the mode used to run the application,
+(Refer to [versioning documentation](manuals/core/versioning/index.md)
+for details) we propose the following migration paths:
+
+The overall principle is to create a version the first time the application
```

Comment:
May be you want to switch these two last lines, so the semicolon introduces the sub-sections.